src/xrpld/core/detail/DatabaseCon.cpp

```diff
@@ -237,6 +237,55 @@ setup_DatabaseCon(Config const& c, std::optional<beast::Journal> j)
     }
     setup.useGlobalPragma = true;
 
+    auto setPragma =
+        [](std::string& pragma, std::string const& key, int64_t value) {
+            pragma = "PRAGMA " + key + "=" + std::to_string(value) + ";";
+        };
+
+    auto isPageSizeValid = [](int64_t page_size) {
```

Comment:
this lambda is not needed, the condition should just be an inline `if` statement